module/userExtension/validator/v1.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (req) => {
+
+    let userExtensionValidator = {
+
+        create: function () {
+            req.checkBody('firstName').exists().withMessage("required firstName");
```

Comment:
@rakeshSgr - I would suggest put here alpha only and other validations. More possible validations you can find here - https://github.com/validatorjs/validator.js#validators, https://express-validator.github.io/docs/validation-chain-api.html and also write custom validations to match password and confirm password etc.

Reply:
we are not using confirm password.